index.js

```diff
@@ -1188,6 +1194,11 @@ module.exports = class Combobo {
       option.disabled = disabled;
       option.innerHTML = label || text;
       option.classList.add(className);
+      if (dataset) {
+        Object.keys(dataset).forEach(key => {
```

Comment:
@stephymiehle This loop is right but you could also use `Object.assign(option.dataset, dataset);` or even `option.dataset = {...option.dataset, ...dataset};` instead.